main/uwsgi/APKBUILD

```diff
@@ -9,14 +9,18 @@ pkgdesc="uWSGI application container server"
 url="http://projects.unbit.it/uwsgi/"
 arch="all"
 license="GPL2"
-depends="mailcap"
+depends="
+        mailcap
+        jansson
```

Comment:
I think that abuild automatically detects this dependency on base of dynamic linking, so it’s unnecessary (nor desirable) to add it here. However, I haven’t verified it (yet).